lib/ae_test_coverage/collectors/webpacker/helpers.rb

```diff
@@ -4,12 +4,12 @@ module AeTestCoverage
   module Collectors
     module Webpacker
       module Helpers
-        def javascript_pack_tag(*names, **options)
```

Comment:
I'm not up to speed with what shakapacker is.  Seems like it is a newer fork of webpacker maybe?  

Did you consider creating a new collector for shakapacker vs modifying the collector for webpacker?

Reply:
Shakapacker is the replacement for webpacker v6. For all intents and purposes, it is webpacker v6 as it still uses all the webpacker namespaces.
Although it might've been better to add this for backwards compatibility potentially, but then again it seems that all the more recent changes are solely targeted at webpacker.